### PR TITLE
style(uploadmanager): deja blue reskin for upload manager

### PR DESCRIPTION
**Pull request by il10**
**BEFORE: **
Screen Shot 2019-10-21 at 4 18 56 PM
Screen Shot 2019-10-21 at 4 20 33 PM

**AFTER: **
Screen Shot 2019-10-21 at 4 19 44 PM
Screen Shot 2019-10-21 at 4 20 05 PM

https://app.zeplin.io/project/5d813c56b87bf954d7b9074e/screen/5daa5ed27746247997b10d41

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @il10 has signed the CLA. Thanks for the pull request!

**Comment by priyajeet** [comment]
Could you add a before/after screenshot?

File: src/elements/content-uploader/UploadsManager.scss [hunk 1]
```diff
@@ -2,9 +2,14 @@
 
 .be.bcu-uploads-manager-container {
```
Comment (shikharmohan) [inline src/elements/content-uploader/UploadsManager.scss]: So you're going to want to scope these changes for deja blue specifically and shouldn't affect non-deja blue webapp. 

So something like this is what you're going to need to do at the end of this file: 

```
.Body.is-deja-blue {
  .be.bcu-uploads-manager-container  {
 // add new styles here
}
}
```

File: src/elements/content-uploader/UploadsManager.scss [hunk 2]
```diff
@@ -2,9 +2,14 @@
...
     position: fixed;
+    right: 56px;
     bottom: 0;
-    width: 100%;
+    left: 288px;
```
Comment (shikharmohan) [inline src/elements/content-uploader/UploadsManager.scss]: This needs to accomodate the scenario when the left nav is collapsed in a responsive mode. So the actual left positioning of the uploads manager should be $left-nav-width + 32px

$left-nav-width can either be 256px or the smaller/collapsed state (don't know the px off the top of my head)

You'll probably need to import the media queries and variables for left nav collapsed state and set the styles accordingly

File: src/elements/content-uploader/UploadsManager.scss [hunk 3]
```diff
@@ -2,9 +2,14 @@
 
 .be.bcu-uploads-manager-container {
     position: fixed;
+    right: 56px;
```
Comment (priyajeet) [inline src/elements/content-uploader/UploadsManager.scss]: Are these positioning styles supposed to be here or in the parent app? Only generic styles should be here ideally.
And for the box-shadow, can you use the $black variable instead. rgba( $black, .1 )

File: src/elements/content-uploader/UploadsManager.scss (hunk 2 above)
Comment (jstoffan) [reply src/elements/content-uploader/UploadsManager.scss]: These styles should live in the parent application, as this component is used by multiple parties, many of whom do not have a left nav at all.

File: src/elements/content-uploader/UploadsManager.scss (hunk 3 above)
Comment (il10) [reply src/elements/content-uploader/UploadsManager.scss]: My bad. All changes go under a deja blue class.

File: src/elements/content-uploader/UploadsManager.scss [hunk 4]
```diff
@@ -2,9 +2,14 @@
...
     bottom: 0;
-    width: 100%;
+    left: 288px;
+    width: unset;
     max-height: 0;
+    border-top-left-radius: 12px;
```
Comment (mickr) [inline src/elements/content-uploader/UploadsManager.scss]: Can these also use a variable as I can imagine that `border-radius` should be consistent ?

Comment (il10) [reply src/elements/content-uploader/UploadsManager.scss]: The radius is only applied to this specific case - it won't be used anywhere else

**Pull request closed without merging**